opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -45,17 +44,14 @@ pub struct LoggerProvider {
     inner: Arc<LoggerProviderInner>,
 }
 
-/// Default logger name if empty string is provided.
-const DEFAULT_COMPONENT_NAME: &str = "rust.opentelemetry.io/sdk/logger";
-
 impl opentelemetry::logs::LoggerProvider for LoggerProvider {
     type Logger = Logger;
 
     fn logger(&self, name: impl Into<Cow<'static, str>>) -> Self::Logger {
-        let mut name = name.into();
+        let name = name.into();
 
         if name.is_empty() {
-            name = Cow::Borrowed(DEFAULT_COMPONENT_NAME)
+            otel_info!(name: "LoggerNameEmpty",  message = "Logger name is empty; consider providing a meaningful name. Logger will function normally and the provided name will be used as-is.");
```

Comment:
lets remove this , otherwise we'll print duplicate logs, one from here and another from inside the logger_with_scope too which this calls.

Reply:
oh yes, missed that.